### PR TITLE
docs: fix typo, tweak path to appium-webdriveragent, add ios-device and iproxy [skip ci]

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Our path to WDA will change in the next release.
In this PR, I changed:

- the path to WDA
- address appium-ios-driver and iproxy
    - As this page's example, change to address iproxy as an alternative way to manage WDA outside Appium (appium-ios-device)

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/advanced-concepts/wda-custom-server.md [hunk 1]
```diff
@@ -61,11 +57,17 @@ so you'll see the icon of WebDriverAgentRunner application on the springboard.
...
 for your application under test. The server address will be _localhost_ if you run your tests on Simulator
-or the actual phone IP address in case of real device. We use _iproxy_ to route network
-requests to a real device from _localhost_ via USB, which means one can use this tool to unify
-WDA network address for Simulator and for real device.
+or the actual phone IP address in case of real device.
+We use [appium-ios-device](https://github.com/appium/appium-ios-device) to route network requests
+to a real device from _localhost_ via USB, which means one can use this tool to unify WDA network
+address for Simulator and for real device.
+
+Alternatively, we can use _iproxy_ to handle WebDriverAgent outside Appium
```
Comment (umutuzgur) [inline docs/en/advanced-concepts/wda-custom-server.md]: @KazuCocoa What do you mean there? We don't use any iproxy in the xcuitest driver right now

Comment (KazuCocoa) [reply docs/en/advanced-concepts/wda-custom-server.md]: This page has an example to build WDA and install it outside Appium create session. So, I leave iproxxy for the following example Java cpde.
Appium connects to the WDA as `webDriverAgentUrl` capability.

Comment (umutuzgur) [reply docs/en/advanced-concepts/wda-custom-server.md]: Gotcha. We can also just connect to a remote device over `appium-ios-device`

**Review by dpgraham — APPROVED** [review]
Nice! Thanks for staying on top of this stuff @KazuCocoa

**Pull request merged**